tests/sai_qualify/sai_infra.py

```diff
@@ -183,6 +183,7 @@ def run_case_from_ptf(duthost, dut_ip, ptfhost, test_case, test_interface_params
     test_para = None
     if request.config.option.enable_ptf_sai_test:
         cmds.append("echo \"export PLATFORM={}\" >> ~/.bashrc".format(get_sai_running_vendor_id(duthost)))
```

Comment:
meanwhile, how about split all those test environment setup, parameter generating into some other shell/script, whatever the language, python, bash or golang. but seperate those SAI-PTF related operation from sonic-mgmt and make SAI repo with SAI-PTF as a whole